Python/Product/ExternalProfilerDriver/DWJsonModel.cs

```diff
@@ -0,0 +1,165 @@
+using System;
```

Comment:
We'll need a header comment with license information on each source file. Apache 2.0 is preferred.